.github/workflows/user_project_ci.yml

```diff
@@ -177,6 +149,9 @@ jobs:
       - name: Export PDK ROOT
         run: echo "PDK_ROOT=/home/runner/work/pdk" >> $GITHUB_ENV
 
+      - name: Export PDK PATH
+        run: echo "PDKPATH=/home/runner/work/pdk/sky130B" >> $GITHUB_ENV
```

Comment:
out of interest, why keep doing this and not set it in the envionment section of the workflow?